.flake8

```diff
@@ -15,3 +23,4 @@ exclude =
   venv*/
   target
   __pycache__
+  */build/lib/*
```

Comment:
Why are we flaking a folder that's not in the source code?

Reply:
This is an exclude list so it is naming folders to ignore when using flake8. The `*/build/lib/*` regex corresponds to ignoring the `build` path that is created when you make a distribution (like when you are releasing).